cypress/elements/fileMenu/open.js

```diff
@@ -2,21 +2,21 @@ import { generateRandomChar, generateRandomNumber } from '../../utils/random'
 import { clickMenuBarFileButton } from '../menuBar'
 import { FILE_MENU_BUTTON_OPEN, clickFileMenuButton } from '.'
 
-const openModalEl = '*[class^="MuiDialogContent"]' // TODO: Add data-test to open modal to target this better
+const openModalEl = 'open-file-dialog-modal'
 const openModalFooterEl = '*[class^="MuiTableFooter"]'
 const openModalToolbarEl = '*[class^="MuiToolbar-root"]'
 const createdByOthersEl = '[data-value=byothers]'
 const openModalItemContainerEl = '*[class^="MuiTableBody"]'
 
 const searchAOByName = name =>
-    cy.get(openModalEl).find('*[type="search"]').clear().type(name)
+    cy.getBySel(openModalEl).find('*[type="search"]').clear().type(name)
 
 const clickRandomAO = () =>
     cy
-        .get(openModalEl)
+        .getBySel(openModalEl)
         .find(openModalItemContainerEl)
         .children()
-        .eq(generateRandomNumber(0, 9))
+        .eq(generateRandomNumber(0, 8))
```

Comment:
```suggestion
        .eq(generateRandomNumber(0, 7))
```
There are only 8 AOs, right, so 0-7 = 8 indexes